tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/OrganisationsSteps.cs

```diff
@@ -123,6 +124,22 @@ public async Task WhenAPUTRequestIsMadeForAnOrganisationWithNameOrganisation(str
             _response.Result = await client.PutAsync(new Uri($"{_organisationUrl}/{organisationId}"), content);
         }
 
+        [When(@"a POST request is made to update an organisation with values")]
+        public async Task WhenAPOSTRequestIsMadeForAnOrganisationWithValues(Table table)
+        {
+            var data = table.CreateInstance<OrganisationTable>();
+
+            using var client = new HttpClient();
+            client.SetBearerToken(_context.Get(ScenarioContextKeys.AccessToken, ""));
+
+            var payload = JsonConvert.SerializeObject(TransformOrganisationIntoPayload(data));
+            using var content = new StringContent(payload, Encoding.UTF8, "application/json");
+            _response.Result = await client.PostAsync(new Uri($"{_organisationUrl}"), content);
+
+            if (_response.Result.StatusCode == HttpStatusCode.Created)
```

Comment:
It probably doesn't matter but I was thinking for consistency reasons we would try to use the `Microsoft.AspNetCore.Http.StatusCodes` instead of the `System.Net` ones e.g. `Status201Created`

Reply:
👍 Consistency rules

Reply:
Wow that is a shame. Yeah OK, apologies I didn't think it would that much trouble.